template/src/index.css

```diff
@@ -25,43 +26,32 @@ body {
     flex-grow: 1;
 }
 
-/* via https://google-webfonts-helper.herokuapp.com/fonts/roboto?subsets=latin */
+/* via https://gwfh.mranftl.com/fonts */
 
-/* roboto-300 - latin */
+/* open-sans-regular - latin */
 @font-face {
-    font-family: "Roboto";
-    font-style: normal;
-    font-weight: 300;
-    src: url("./assets/fonts/roboto-v15-latin-300.eot"); /* IE9 Compat Modes */
-    src: local("Roboto Light"), local("Roboto-Light"),
-        url("./assets/fonts/roboto-v15-latin-300.eot?#iefix") format("embedded-opentype"),
-        /* IE6-IE8 */ url("./assets/fonts/roboto-v15-latin-300.woff2") format("woff2"),
-        /* Super Modern Browsers */ url("./assets/fonts/roboto-v15-latin-300.woff") format("woff"),
-        /* Modern Browsers */ url("./assets/fonts/roboto-v15-latin-300.svg#Roboto") format("svg"); /* Legacy iOS */
-}
-
-/* roboto-regular - latin */
-@font-face {
-    font-family: "Roboto";
+    font-family: 'Open Sans';
     font-style: normal;
     font-weight: 400;
-    src: url("./assets/fonts/roboto-v15-latin-regular.eot"); /* IE9 Compat Modes */
-    src: local("Roboto"), local("Roboto-Regular"),
-        url("./assets/fonts/roboto-v15-latin-regular.eot?#iefix") format("embedded-opentype"),
-        /* IE6-IE8 */ url("./assets/fonts/roboto-v15-latin-regular.woff2") format("woff2"),
-        /* Super Modern Browsers */ url("./assets/fonts/roboto-v15-latin-regular.woff") format("woff"),
-        /* Modern Browsers */ url("./assets/fonts/roboto-v15-latin-regular.svg#Roboto") format("svg"); /* Legacy iOS */
+    src: url('./assets/fonts/open-sans-v34-latin-regular.eot'); /* IE9 Compat Modes */
+    src: local(''),
```

Comment:
remove `local('')`, shouldn't be needed anymore and the `local('☺')` [sic!] dates back to <IE8 times.
eventually furthermore add `font-display: swap;`

related, both unfixed in upstream gwfh: 
* https://github.com/majodev/google-webfonts-helper/issues/84
* https://github.com/majodev/google-webfonts-helper/pull/123